### PR TITLE
Added general velocity tools

### PR DESCRIPTION
**Pull request by ryan-lam**
Added general velocity tools, docs, tests

- function to change speed, bearing, elevation to vectors
- function to project vectors

### CODE REVIEW

**Review by probro27 — COMMENTED** [review]
Hey @ryan-lam this looks pretty good. Though I am not sure, is there anywhere we want to use such a vector? Or is this just to help us visualize things?
Just want to understand the need of this as I am not currently sure where we would use this?
Thanks :)

**Comment by ryan-lam** [comment]
Yea, it'll be used. I'm still working on the code that uses it. It's in another branch (https://github.com/uw-midsun/strategy_xv/blob/car-forces/routemodel/carforces/carforces.py) to do vector math and calculate vector projections

It's used in calculating the force on the car and any code related to velocities (ie. wind speed)

File: routemodel/tools/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Tools.py: General tools that may be useful throughout dev
```
Comment (probro27) [inline routemodel/tools/README.md:1]: Please add some more documentation as to how the vector looks and what we are returning. 
Provide a couple of examples with calculations (if possible).

**Review by probro27 — CHANGES_REQUESTED** [review]
Cool, I just want you to update the documentation a bit for this. The rest looks amazing and I can see why you would need this. Thanks a lot. 
Great job!

**Review by probro27 — APPROVED** [review]
LGTM!
Great job @ryan-lam. 
Merging it in :)

**Pull request merged**